tests/e2e/mgmt_wrkld_domain_isolation.go

```diff
@@ -130,6 +142,45 @@ var _ bool = ginkgo.Describe("[domain-isolation] Management-Workload-Domain-Isol
 		} else {
 			sharedStorageProfileId = e2eVSphere.GetSpbmPolicyID(sharedStoragePolicyName)
 		}
+
+		// fetch shared vsphere datatsore url
+		datastoreURL := GetAndExpectStringEnvVar(envSharedDatastoreURL)
+		if datastoreURL == "" {
```

Comment:
This may cause entire files to get skipped.
Please  use it this way in test case, not in before each. 
datastoreURL := os.Getenv("<SHARED_ENV>")
		if datastoreURL == "" {
			ginkgo.Skip("Env SHARED_ENV is missing")
		}